configs/fmri_sc_seg-2d.json

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
change already in https://github.com/sct-pipeline/fmri-segmentation/pull/12/files